controllers/argocd/configmap.go

```diff
@@ -178,6 +178,13 @@ func getResourceInclusions(cr *argoprojv1a1.ArgoCD) string {
 	return re
 }
 
+// getResourceTrackingMethod will return the resource tracking method for the given ArgoCD.
+func getResourceTrackingMethod(cr *argoprojv1a1.ArgoCD) string {
+	rtm := argoprojv1a1.ParseResourceTrackingMethod(cr.Spec.ResourceTrackingMethod)
+	log.Info(fmt.Sprintf("Found %s as tracking method", cr.Spec.ResourceTrackingMethod))
```

Comment:
Would it make sense to log an error if the tracking method was invalid?  Not sure the user would ever see it, though.

Reply:
Yeah, I agree that we should probably log it out, as this is good practice. Thanks!

Reply:
Done.